config/manager/kustomization.yaml

```diff
@@ -1,2 +1,2 @@
 resources:
-- manager.yaml
+- manager.yaml
```

Comment:
This looks like an unnecessary change. Otherwise LGTM.

Reply:
Updated